Services/PrivacyConsentService.cs

```diff
@@ -38,15 +39,11 @@ public async Task<bool> IsConsentBannerNeededAsync(HttpContext httpContext)
             if (httpContext.User.Identity.IsAuthenticated)
             {
                 var user = await _userService.GetAuthenticatedUserAsync(httpContext.User);
-
                 return user is not User orchardUser || !orchardUser.Has<PrivacyConsent>();
             }
-            else
-            {
-                var cookieConsent = httpContext.Request.Cookies[_cookiePolicyOptions.Value.ConsentCookie.Name];
 
-                return cookieConsent is null;
-            }
+            var cookieConsent = httpContext.Request.Cookies[_cookiePolicyOptions.Value.ConsentCookie.Name];
+            return cookieConsent is null;
```

Comment:
Not related, but weren't we against `is null` and in favor of `== null`?

Reply:
Do you keep record of conventions like this anywhere?

Reply:
Yeah, this is just needlessly like Basic.

I guess Dávid is referring to a convo under a PR you've also taken part in. Otherwise, these are usually on Teams.

Reply:
Ah, I faintly remember that. Thanks.

Reply:
Basic does that too? It reminds me of Python.

It was called out in one of my PRs in the past and also there was [this convo on teams](https://teams.microsoft.com/l/message/19:525a1a3656c845ae92c5e2c0d3617cc7@thread.skype/1623235688771?tenantId=959ede5a-38e9-44e6-adc8-85553beb8220&groupId=d6a5d775-6fb9-41dc-a7bb-481d4f884046&parentMessageId=1623235688771&teamName=Lombiq%20Developers&channelName=General&createdTime=1623235688771) by Oliver as well.

By the way I've recently found out about this while working on HAST, there is a legitimate difference here. When you compile c# your `value is null` becomes `((object)value) == null` which means that is bypasses any custom equation operator overloading you may have.  This too is similar to how `is` works in Python. It's a pure reference check.

Reply:
Ah! That's good to know. This should make a difference only in very rare situations, though.